### DISCUSSION TITLE
Can you still buy these pre-assembled?

### DISCUSSION THREAD

**Discussion by synio-wesley** [revk/xESP32-Faikout#287] [Q&A] [answered]
Can you still buy these ESP32-Faikin boards pre-assembled? It looks like they are not available through Amazon anymore. Are they coming back soon?

Otherwise I'm also willing to try buying on JLCPCB but my experience with that is very limited. I can solder and do a little bit of electronics. But I guess it is also possible to have JLCPCB assemble it all? But I'm not sure about the settings. I uploaded the zip file and kept the defaults like this: FR-4, 2 layers, 70x70 mm, 1.6 thickness, HASL with lead surface finish, single PCB delivery format, 1 oz outer copper, tented via covering, and all the other defaults.

Then I guess I should add "PCB Assembly" option? If I select Economic PCBA Type it looks like 2 of the parts can't be added when I later upload the BOM file. So I should choose Standard PCBA then? (and advanced options as defaults?)

With Standard PCBA it seems to find all the parts. Although for the XL-1010RGBC-WS2812B SMD LED there is a note and I don't know what this actually means in practice: "Moisture-sensitive LED, need to be baked / After placing an order, it will be reviewed. If PCBA advanced option is required, additional fees will be charged. The review result will be notified to you by email."

Looks like that would give me 5 assembled boards (without cabling) for a cost of around $ 110 if I'm not mistaken? Does that sounds right, and would this be what I need? I think I'd rather buy 2 of these through Amazon if they are available.

Sorry if this is not a good question. My experience with electronics is limited, and I just want to get Home Assistant working (preferably locally) with 2 Daikin ACs.

**Comment by revk** [comment]
Amazon UK have over 400 in stock.

And yes, JLCPCB can be a pain some times.

**Reply by synio-wesley** [reply]
Huh okay.. that's weird.. Maybe it's because of the shipping then.. I just see this message.. I will look into it and try to find out why it says it is unavailable.

![image](https://github.com/revk/ESP32-Faikin/assets/30413/3c5512d7-8fbd-451a-bd4d-5f35934b0367)

**Reply by revk** [reply]
This one? https://www.amazon.co.uk/dp/B0C2ZYXNYQ

<img width="1433" alt="Screenshot 2024-03-30 at 15 00 43" src="https://github.com/revk/ESP32-Faikin/assets/996983/38d2b689-2d93-4c93-a092-2246b7d13e7b">

**Reply by synio-wesley** [reply]
Yes... that link gives me the "Currently unavailable" message... maybe because I'm in Belgium or something?

**Reply by revk** [reply, marked as answer]
There are shipping companies that can provide a UK address. Sorry it is a pain but I have no way to control Amazon on this - they seem to decide which countries they will "buy and resell" to, which seems quite a lot (Netherlands, Australia, US even some times).

**Reply by synio-wesley** [reply]
Ok yea if I change the address and fill in a UK postcode then I do see it "In stock" sorry for the misunderstanding.. I will try to figure out something.. Thanks! Edit: oh, The Netherlands works too.. I live close by the border so I might find a friend who I can send it to that way..

**Reply by ghost** [reply]
Hey Wesley,
"Een Uitgeweken Belg hier" now living fulltime in the netherlands.  
We have a new Daikin airco installation planned for this week, so I might also need to order these units from the UK soon.
From time to time I do need to drop by Belgium again for some grocery shopping missing from the local supermarkets.

**Reply by synio-wesley** [reply]
Oh ok, interesting. I'm expecting our Daikins to be installed in May or June. But maybe I can already order together with you so we can both save on shipping cost. I'm at work now but if you can send me a message to "wesley (at) syn (dot) io" then maybe we can get in touch about this.
